extensions/wikia/RTE/RTE.class.php

```diff
@@ -682,7 +676,7 @@ static function onEditingPreferencesBefore($user, &$preferences) {
 	/**
 	 * Modify values returned by User::getGlobalPreference() when wysiwyg is enabled
 	 */
-	static public function userGetOption($options, $name, &$value) {
+	static public function userGetOption( $options, $name, &$value ): bool {
 		global $wgRequest;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)